ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp

```diff
@@ -1130,13 +1176,73 @@ void py_module(py::module& module) {
-    detail::bind_unary_composite_threshold_value(module, ttnn::threshold);
+    detail::bind_unary_composite_two_float_with_default(
+        module,
+        ttnn::hardswish,
+        "scale", "Scale value", 1.0f/6.0f,
+        "shift", "Shift value", 0.5f,
+        R"doc(Performs hardswish function on :attr:`input_tensor`, :attr:`scale`, :attr:`shift`.)doc");
+    detail::bind_unary_composite_two_float_with_default(
+        module,
+        ttnn::hardsigmoid,
+        "scale", "Scale value", 1.0f/6.0f,
+        "shift", "Shift value", 0.5f,
+        R"doc(Performs hardsigmoid function on :attr:`input_tensor`, :attr:`scale`, :attr:`shift`.)doc");
+    detail::bind_unary_composite_two_float_with_default(
+        module,
+        ttnn::hardtanh,
+        "min", "min value", -1.0f,
+        "max", "max value", 1.0f,
+        R"doc(Performs hardtanh function on :attr:`input_tensor`, :attr:`min`, :attr:`max`.)doc");
+    detail::bind_unary_composite_two_float(
+        module,
+        ttnn::clip,
+        "low", "Low value",
+        "high", "High value",
+        R"doc(Performs clip function on :attr:`input_tensor`, :attr:`low`, :attr:`high`.)doc");
+    detail::bind_unary_composite_two_float(
+        module,
+        ttnn::clamp,
+        "low", "Low value",
+        "high", "High value",
+        R"doc(Performs clamp function on :attr:`input_tensor`, :attr:`low`, :attr:`high`.)doc");
+    detail::bind_unary_composite_two_float_with_default(
+        module,
+        ttnn::selu,
+        "scale", "Scale value", 1.0507009873554804934193349852946,
+        "alpha", "Alpha value", 1.6732632423543772848170429916717,
+        R"doc(Performs selu function on :attr:`input_tensor`, :attr:`scale`, :attr:`alpha`.)doc");
+    detail::bind_unary_composite_two_float(
+        module,
+        ttnn::threshold,
+        "threshold", "Threshold value",
+        "value", "Value value",
+        R"doc(Performs threshold function on :attr:`input_tensor`, :attr:`threshold`, :attr:`value`.)doc");
+    detail::bind_unary_composite_int_with_default(
+        module,
+        ttnn::tril,
+        "diagonal", "diagonal value", 0,
+        R"doc(Performs tril function on :attr:`input_tensor`, :attr:`diagonal`.)doc");
+    detail::bind_unary_composite_int_with_default(
+        module,
+        ttnn::triu,
+        "diagonal", "diagonal value", 0,
+        R"doc(Performs triu function on :attr:`input_tensor`, :attr:`diagonal`.)doc");
+    detail::bind_unary_composite_int_with_default(
+        module,
+        ttnn::round,
+        "decimals", "decimals value", 0,
+        R"doc(Performs round function on :attr:`input_tensor`, :attr:`decimals`.)doc");
+    detail::bind_unary_composite_one_int(
+        module,
+        ttnn::polygamma,
+        "k", "k value",
+        R"doc(Performs polygamma function on :attr:`input_tensor`, :attr:`decimals`. it is supported for range 1 to 10 only)doc");
+    detail::bind_unary_composite_one_int_bool(
```

Comment:
this is used only for argmin, rename this bind_argmin

Reply:
why should go to case specific. we are trying to make it generalise?